terraform/modules/compute/wls_compute/wls_compute.tf

```diff
@@ -76,6 +76,8 @@ module "wls-instances" {
       keystore_password_id               = var.keystore_password_id
       root_ca_id                         = var.root_ca_id
       thread_pool_limit                  = var.thread_pool_limit
+      wls_admin_user_1                   = var.wls_admin_user_1
+      wls_admin_password_ocid_1          = var.wls_admin_password_id_1
```

Comment:
This is a nitpick (optional fix): Reflects the same change as the other branch variable name.
I would call this wls_admin_password_1_ocid, wls_admin_password_1_id etc. (rather than password_ocid_1)